Week1/homework/js-exercise/dogPhoto.js

```diff
@@ -0,0 +1,42 @@
+const xmlBtn = document.querySelector("#xml");
+const axiosBtn = document.querySelector("#axios");
+const dogs = document.querySelector("#dogs");
```

Comment:
I would suggest to call this `dogs` instead, it's a list of dogs and not a one dog, don't you agree?
```suggestion
const dogs = document.querySelector('#dogs');
```

you need to make few changes in the file afterwords